README.md

```diff
@@ -72,7 +72,7 @@ Finally, create a new `ActionCache` with optional name and expiration within you
   - If an `ttl` is provided, expired cache entries are deleted when they are retrieved and in a daily cron job.
 
 ```ts
-import { ActionCache } from "@convex-dev/cache";
```

Comment:
`ActionCache` is imported from `@convex-dev/action-cache`
`@convex-dev/cache` package does not exist